### DISCUSSION TITLE
Feature Update AMD CPU Uplift 24H2

### DISCUSSION THREAD

**Discussion by TalonRahl** [memstechtips/UnattendedWinstall#123] [General]
Hello, 

Looking at the most recent build of 24H2 it looks like there is an amazing uplift for AMD cpu gaming framerate. I want this update. 
Has anyone tried using the answer file with this new update? 

Dankeschon,

Talon

**Comment by memstechtips** [comment]
I will test the answer file on 24H2 once it's officially released, not while it's in preview.
